sopel/modules/xkcd.py

```diff
@@ -132,3 +132,9 @@ def say_result(bot, result, commanded=True):
 def get_url(bot, trigger, match):
     latest = get_info()
     numbered_result(bot, int(match.group(1)), latest, commanded=False)
+
+
+@url(r'xkcd.com/?$')
```

Comment:
```suggestion
@url(r'https?://xkcd\.com/?$')
```

We don't want to match, for example, `ftp://explainxkcd.com/`. 😛